rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -490,8 +504,6 @@ maas_rpc_scripts_dir: /opt/rpc-openstack/scripts
 #
 maas_excluded_checks: []
 
-
```

Comment:
Still removing these by accident? I recall you telling me this wasn't intentional last time.